### PR TITLE
[Messages] Convert messages from Spells to FocusEffect where necessary.

### PR DESCRIPTION
**Pull request by Kinglykrab**
https://github.com/EQEmu/Server/issues/837 from 2019 notes a couple places that should use Focus Effect chat type instead of Spells chat type.

### CODE REVIEW

**Review by nolangreene — APPROVED** [review]

File: zone/spells.cpp [hunk 1]
```diff
@@ -478,7 +478,7 @@ bool Mob::DoCastSpell(uint16 spell_id, uint16 target_id, CastingSlot slot,
 	if (IsClient() && slot == CastingSlot::Item && item_slot != 0xFFFFFFFF) {
 		auto item = CastToClient()->GetInv().GetItem(item_slot);
 		if (item && item->GetItem())
-			MessageString(Chat::Spells, BEGINS_TO_GLOW, item->GetItem()->Name);
+			MessageString(Chat::FocusEffect, BEGINS_TO_GLOW, item->GetItem()->Name);
```
Comment (hgtw) [inline zone/spells.cpp:481]: Live doesn't have this message (anymore?) when casting items so it should probably be removed or rule gated. Tested with a couple random items (Oil of Fennin Ro and Elder Spiritist Boots)

**Review by hgtw — APPROVED** [review]

File: zone/spells.cpp (hunk 1 above)
Comment (Kinglykrab) [reply zone/spells.cpp:481]: Is this the only message they removed?

Comment (hgtw) [reply zone/spells.cpp:481]: Furious bash and the Client::GetFocusEffect messages still occur. I dunno what merc focus effects are but the function is also `GetFocusEffect` so it's probably ok

**Review by hgtw — APPROVED** [review]

File: zone/spell_effects.cpp [hunk 2]
```diff
@@ -6497,7 +6497,7 @@ int64 Client::GetFocusEffect(focusType type, uint16 spell_id, Mob *caster, bool
 			default:
 				break;
 			}
-			MessageString(Chat::Spells, string_id, UsedItem->Name);
+			MessageString(Chat::FocusEffect, string_id, UsedItem->Name);
```
[the remark this answers is not in the document]
Comment (hgtw) [reply zone/spell_effects.cpp:6500]: Ignore that, it looks like emu already correctly handles focus orders. I must have been testing a focus that occured on cast start

**Pull request merged**